packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -858,3 +876,16 @@ export enum StreamEncryption {
    */
   MANAGED = 'MANAGED'
 }
+
+/**
+ * Specifies the capacity mode to apply to this stream.
+ */
+export enum StreamModeDetails {
```

Comment:
Should we call this just `StreamMode`? The word "details" doesn't add any meaning here.

Reply:
I agree. I just used the CloudFormation terminology because I wasn't sure if it was appropriate to change the name in CDK. I'd be happier with the `StreamMode`.